### PR TITLE
API Libraries > Sidecar REST API

### PR DESCRIPTION
**Pull request by nhussein11**
This page aims to add the Sidecar REST API

### CODE REVIEW

File: develop/toolkit/api-libraries/sidecar.md [hunk 1]
```diff
@@ -1,4 +1,100 @@
...
+    ```
+
+    If you need to install or update Node.js, visit the [official Node.js website](https://nodejs.org/){target=\_blank} to download and install the latest LTS version.
+
+You can confirm the installation by running:
+
+```bash
+substrate-api-sidecar --version
+```
+
```
Comment (0xLucca) [inline develop/toolkit/api-libraries/sidecar.md:57]: Add a reference to the official installation guide: https://github.com/paritytech/substrate-api-sidecar?tab=readme-ov-file#npm-package-installation-and-usage

File: develop/toolkit/api-libraries/sidecar.md [hunk 2]
```diff
@@ -1,4 +1,100 @@
...
+## Usage
+
+To use the Sidecar API, you have two options:
+
+- Local node - run a Substrate node locally, which Sidecar will connect to by default, requiring no additional configuration. To start, run:
```
Comment (0xLucca) [inline develop/toolkit/api-libraries/sidecar.md:62]:
```suggestion
- Local node - run a node locally, which Sidecar will connect to by default, requiring no additional configuration. To start, run:
```

**Review by 0xLucca — APPROVED** [review]
LGTM

**Comment by eshaben** [comment]
@themacexpert can you please do a formatting review on this one?

File: develop/toolkit/api-libraries/sidecar.md [hunk 3]
```diff
@@ -1,4 +1,100 @@
...
+
+## Introduction
+
+The [Sidecar Rest API](https://github.com/paritytech/substrate-api-sidecar){target=\_blank} is a service that provides a REST interface for interacting with Polkadot SDK-based blockchains. With this API, developers can easily access a broad range of endpoints for nodes, accounts, transactions, parachains, and more.
+
+Sidecar functions as a caching layer between your application and a Polkadot SDK-based node, offering standardized REST endpoints that simplify interactions without requiring complex, direct RPC calls. This approach is especially valuable for developers who prefer REST APIs or build applications in languages with limited WebSocket support.
+
+Some of the key features of the Sidecar API include:
+
+- **REST API Interface** - provides a familiar REST API interface for interacting with Polkadot SDK-based chains
```
Comment (eshaben) [inline develop/toolkit/api-libraries/sidecar.md:16]: based on a quick look, for the list items here. on the left, only the first word needs to be capitalized

File: develop/toolkit/api-libraries/sidecar.md [hunk 4]
```diff
@@ -1,4 +1,100 @@
...
+
+```bash
+substrate-api-sidecar --version
+```
+
+## Usage
+
+To use the Sidecar API, you have two options:
+
+- Local node - run a node locally, which Sidecar will connect to by default, requiring no additional configuration. To start, run:
```
Comment (themacexpert) [inline develop/toolkit/api-libraries/sidecar.md:62]:
```suggestion
- **Local node** - run a node locally, which Sidecar will connect to by default, requiring no additional configuration. To start, run:
```

File: develop/toolkit/api-libraries/sidecar.md [hunk 5]
```diff
@@ -1,4 +1,100 @@
...
+    ```
+    substrate-api-sidecar
+    ```
+- Remote Node - connect Sidecar to a remote node by specifying the RPC endpoint for that chain. For example, to gain access to the Polkadot Asset Hub associated endpoints:
```
Comment (themacexpert) [inline develop/toolkit/api-libraries/sidecar.md:66]:
```suggestion
- **Remote node** - connect Sidecar to a remote node by specifying the RPC endpoint for that chain. For example, to gain access to the Polkadot Asset Hub associated endpoints:
```

File: develop/toolkit/api-libraries/sidecar.md [hunk 6]
```diff
@@ -1,4 +1,100 @@
...
+
+In response, you’ll see output similar to this (assuming you’re connected to Polkadot Asset Hub):
+
+--8<-- 'code/develop/toolkit/api-libraries/sidecar/asset-hub-node-version-response.md'
+
+For a complete list of available endpoints and their documentation, visit the [Sidecar API list endpoints](https://paritytech.github.io/substrate-api-sidecar/dist/){target=\_blank}. You can learn about the endpoints and how to use them in your applications.
+
+## Next Steps
+
+To dive deeper, refer to the [official Sidecar documentation](https://github.com/paritytech/substrate-api-sidecar?tab=readme-ov-file#substrateapi-sidecar){target=\_blank}. This will provide a comprehensive guide to the available configurations and advanced usage.
```
Comment (themacexpert) [inline develop/toolkit/api-libraries/sidecar.md:100]:
```suggestion
To dive deeper, refer to the [official Sidecar documentation](https://github.com/paritytech/substrate-api-sidecar?tab=readme-ov-file#substrateapi-sidecar){target=\_blank}. This provides a comprehensive guide to the available configurations and advanced usage.
```

File: develop/toolkit/api-libraries/sidecar.md (hunk 3 above)
Comment (nhussein11) [reply develop/toolkit/api-libraries/sidecar.md]: fixed on [9afc1a8](https://github.com/polkadot-developers/polkadot-docs/pull/141/commits/9afc1a867dc554a740304603430fc6c792acc5c1)! Thank you!!

File: develop/toolkit/api-libraries/sidecar.md [hunk 7]
```diff
@@ -1,4 +1,102 @@
...
+!!! note
+    Alternatively, you can access `http://127.0.0.1:8080/node/version` directly in a browser since it’s a `GET` request.
+
+In response, you’ll see output similar to this (assuming you’re connected to Polkadot Asset Hub):
+
+--8<-- 'code/develop/toolkit/api-libraries/sidecar/asset-hub-node-version-response.md'
+
+For a complete list of available endpoints and their documentation, visit the [Sidecar API list endpoints](https://paritytech.github.io/substrate-api-sidecar/dist/){target=\_blank}. You can learn about the endpoints and how to use them in your applications.
+
+## Next Steps
```
Comment (eshaben) [inline develop/toolkit/api-libraries/sidecar.md:100]:
```suggestion
## Where to Go Next
```

**Review by eshaben — COMMENTED** [review]
One last thing then this will be good to go!

**Comment by nhussein11** [comment]
@themacexpert I think that, with your approval, this PR would be good to merge :) 🚀

**Review by eshaben — APPROVED** [review]
LGTM!

**Pull request merged**